packages/react-router/src/hooks/redirect.ts

```diff
@@ -3,10 +3,10 @@ import type {NavigateTo} from '../types';
 
 import {useNavigate} from './navigate';
 
-export function useRedirect(to: NavigateTo) {
+export function useRedirect(to: NavigateTo, relativeTo?: string) {
```

Comment:
I'd like the second option to be an options object so it matches the `navigate` signature, what do you think of that?

Reply:
Do you mean have something like that: ?

```suggestion
export function useRedirect(to: NavigateTo, {replace, relativeTo} = {replace = false}) {
```

Reply:
Done (I think ;))